tests/api/test_controller.py

```diff
@@ -57,6 +64,10 @@ def setup_method(self):
             self.set_base_url(self._db)
             app.manager = self.circulation_manager_setup(self._db)
 
+    def teardown_method(self):
+        self.patch.undo()
+        super().teardown_method()
+
```

Comment:
This code should be able to go, once these tests move over to the new fixture system.